packages/cli/scripts/preinstall.js

```diff
@@ -1,21 +1,40 @@
+const process = require('node:process')
 // @ts-check
 const { drawBox } = require('@prisma/internals/dist/utils/drawBox')
 
 export function main() {
-  // process.version (e.g. `v16.0.0`)
-  printMessageAndExitIfUnsupportedNodeVersion(process.version)
+  const nodeVersion = process.version.slice(1)
+
+  printMessageAndExitIfUnsupportedNodeVersion(nodeVersion)
```

Comment:
```suggestion
  printMessageAndExitIfUnsupportedNodeVersion(process.version)
```
Pass the version directly and avoid slicing the first character twice

Reply:
Thanks, this was a leftover